docs/rest-api.md

```diff
@@ -0,0 +1,909 @@
+    <td colspan="3">1</td>
+  </tr>
+  <tr>
+    <th rowspan="2">Result Columns</th>
+    <th>Column Name</th>
+    <th>Column Type</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>job_id</td>
+    <td>varchar</td>
+    <td>By executing a <code>INSERT</code> statement we mean to submit the statement to the cluster. This column contains the ID of the generated Flink job. To fetch the result of this job, see "Fetching the Result" section.</td>
+  </tr>
+</table>
+
+#### DDL
+
+<table>
+  <tr>
+    <th colspan="2">Statement Type</th>
+    <th colspan="2">Example</th>
+  </tr>
+  <tr>
+    <td colspan="2">CREATE_TABLE</td>
+    <td colspan="2"><code>CREATE TABLE t( a INT, b BIGINT )</code></td>
+  </tr>
+  <tr>
+    <td colspan="2">DROP_TABLE</td>
+    <td colspan="2"><code>DROP TABLE t</code></td>
+  </tr>
+  <tr>
+    <td colspan="2">ALTER_TABLE</td>
+    <td colspan="2"><code>ALTER TABLE t1 RENAME TO t2</code></td>
+  </tr>
+  <tr>
+    <td colspan="2">CREATE_VIEW</td>
+    <td colspan="2"><code>CREATE VIEW myview AS SELECT * FROM t</code></td>
+  </tr>
+  <tr>
+    <td colspan="2">DROP_VIEW</td>
+    <td colspan="2"><code>DROP VIEW myview</code></td>
+  </tr>
+  <tr>
+    <td colspan="2">CREATE_DATABASE</td>
+    <td colspan="2"><code>CREATE DATABASE mydb</code></td>
+  </tr>
+  <tr>
+    <td colspan="2">DROP_DATABASE</td>
+    <td colspan="2"><code>DROP DATABASE mydb</code></td>
+  </tr>
+  <tr>
+    <td colspan="2">ALTER_DATABASE</td>
+    <td colspan="2"><code>ALTER DATABASE mydb SET ('k1' = 'a')</code></td>
+  </tr>
+  <tr>
+    <th>Result Kind</th>
+    <td colspan="3">SUCCESS</td>
+  </tr>
+  <tr>
+    <th>Row Count</th>
+    <td colspan="3">1</td>
+  </tr>
+  <tr>
+    <th rowspan="2">Result Columns</th>
+    <th>Column Name</th>
+    <th>Column Type</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>result</td>
+    <td>varchar</td>
+    <td>Must be <code>OK</code>.</td>
+  </tr>
+</table>
+
+#### USE
+
+<table>
+  <tr>
+    <th colspan="2">Statement Type</th>
+    <th colspan="2">Example</th>
+  </tr>
+  <tr>
+    <td colspan="2">USE_CATALOG</td>
+    <td colspan="2"><code>USE CATALOG mycatalog</code></td>
+  </tr>
+  <tr>
+    <td colspan="2">USE</td>
+    <td colspan="2"><code>USE mydb</code></td>
+  </tr>
+  <tr>
+    <th>Result Kind</th>
+    <td colspan="3">SUCCESS</td>
+  </tr>
+  <tr>
+    <th>Row Count</th>
+    <td colspan="3">1</td>
+  </tr>
+  <tr>
+    <th rowspan="2">Result Columns</th>
+    <th>Column Name</th>
+    <th>Column Type</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>result</td>
+    <td>varchar</td>
+    <td>Must be <code>OK</code>.</td>
+  </tr>
+</table>
+
+#### SHOW CATALOGS
+
+<table>
+  <tr>
+    <th>Statement Type</th>
+    <td>SHOW_CATALOGS</td>
+    <th>Example</th>
+    <td><code>SHOW CATALOGS</code></td>
+  </tr>
+  <tr>
+    <th>Result Kind</th>
+    <td colspan="3">SUCCESS_WITH_CONTENT</td>
+  </tr>
+  <tr>
+    <th>Row Count</th>
+    <td colspan="3">&ge;1</td>
```

Comment:
catalogs may be empty

Reply:
catalogs will always be non empty, but tables and views can indeed be empty